src/components/FocusModeNotification.js

```diff
@@ -0,0 +1,47 @@
+import React, {useEffect} from 'react';
+import useEnvironment from '@hooks/useEnvironment';
+import useLocalize from '@hooks/useLocalize';
+import styles from '@styles/styles';
+import * as Link from '@userActions/Link';
+import * as User from '@userActions/User';
+import CONST from '@src/CONST';
+import ConfirmModal from './ConfirmModal';
+import Text from './Text';
+import TextLinkWithRef from './TextLink';
+
+function FocusModeNotification() {
+    const {environmentURL} = useEnvironment();
+    const {translate} = useLocalize();
+    useEffect(() => {
+        User.updateChatPriorityMode(CONST.PRIORITY_MODE.GSD, true);
+    }, []);
+    const href = `${environmentURL}/settings/preferences/priority-mode`;
+    return (
+        <ConfirmModal
+            title={translate('focusModeUpdateModal.title')}
+            confirmText={translate('common.buttonConfirm')}
+            onConfirm={User.clearFocusModeNotification}
+            shouldShowCancelButton={false}
+            prompt={
+                <Text>
+                    {translate('focusModeUpdateModal.prompt')}
+                    <TextLinkWithRef
+                        href={href}
+                        style={styles.link}
+                        onPress={() => {
+                            User.clearFocusModeNotification();
+                            Link.openLink(href, environmentURL);
```

Comment:
No real reason other than I wanted to refactor and centralize this logic so that it would work with "deep links".  I guess you are right that we could have used a path in this case. But I won't change it.